### ISSUE TITLE
Draggables not working for heavy handed touch

### ISSUE THREAD

**Issue by markbrown4** [atlassian/react-beautiful-dnd#1021]
## Bug

A simple vertical list with sortable div's does not work reliably on touch devices, (tested on iPhone6).
Only when you touch lightly and hold does it work, if you're too heavy handed you can't use the Draggables at all.

The demo example shows this behaviour:
https://codesandbox.io/s/k260nyxq9v

### Expected behavior

For simple lists without links you should be able to drag on the handle reliably every time, regardless of pressure.

I believe this is related to the force touch support that was added for links, when there's no links and no need for force touch I'm hoping this can improved, otherwise we're going to need to fallback to up / down buttons on mobile, it doesn't work enough of the time and makes for a worse experience.

**Versions:**
`react`: 16.6.3
`react-beautiful-dnd`: 10.0.3

**Browsers:**
Tested on ios 12.1.1:

- Mobile Safari
- Chrome
- Firefox

**Comment by alexreardon** [comment]
Here is the relevant code:

```js
// Need to opt out of dragging if the user is a force press
    // Only for webkit which has decided to introduce its own custom way of doing things
    // https://developer.apple.com/library/content/documentation/AppleApplications/Conceptual/SafariJSProgTopics/RespondingtoForceTouchEventsfromJavaScript.html
    {
      eventName: 'touchforcechange',
      fn: (event: TouchEvent) => {
        // A force push action will no longer fire after a touchmove
        if (state.hasMoved) {
          // This is being super safe. While this situation should not occur we
          // are still expressing that we want to opt out of force pressing
          event.preventDefault();
          return;
        }

        // A drag could be pending or has already started but no movement has occurred

        const touch: TouchWithForce = (event.touches[0]: any);

        if (touch.force >= forcePressThreshold) {
          // this is an indirect cancel so we do not preventDefault
          // we also want to allow the force press to occur
          cancel();
        }
      },
    },
```

Right now we cancel a drag if we detect a force press. 

Here is the relevant section from the README:

> ### Understanding intention: tap, force press, scroll and drag
> 
> When a user presses their finger (or other input) on a `Draggable` we are not sure if they where intending to _tap_, _force press_, _scroll the container_ or _drag_. **As much as possible `react-beautiful-dnd` aims to ensure that a users default interaction experience remains unaffected**.
> To see more indepth information about how we impact standard browser events see our [how we use DOM events guide](/docs/guides/how-we-use-dom-events.md)

In order to respect force touch interactions we opt out of a touch drag starting if we detect a force press

**Comment by alexreardon** [comment]
To your point though, this can leave only a small window of pressure for dragging.

**Comment by alexreardon** [comment]
Perhaps we could only apply the logic for certain tag types, or make it a prop? 🤔

**Comment by markbrown4** [comment]
Yep, removing that code makes it work consistently for me, I'll fork for now.

I believe force touch is only relevant when the Draggables are links, so there should be a way to opt in to that extra code when needed.  It makes it unusable in the simple case.

**Comment by alexreardon** [comment]
We could look at setting an allow list of tag names where we do the force press detection.

Alternatively, we could have an `overrideForcePress` (or similarly named) prop

**Comment by markbrown4** [comment]
@levibuzolic has an example of making this force push handling opt-out with a prop.

https://github.com/atlassian/react-beautiful-dnd/compare/master...levibuzolic:canCancelDragWithForceTouch?expand=1

Would you be open to adding a prop like this?  The future of force touch is shaky so I don't know if you want to add it to your API.

**Comment by alexreardon** [comment]
I wonder if an allow list based on tag would be a better path forward..

**Comment by levibuzolic** [comment]
> I wonder if an allow list based on tag would be a better path forward..

That could work, though wouldn't that potentially mean that custom force touch elements wouldn't work?

Another alternative would be to allow the `forcePressThreshold` to be configured directly. 🤔

**Comment by alexreardon** [comment]
I think it better to keep a boolean if we where to go down the prop approach given that people will want to be either opting in or out

**Comment by alexreardon** [comment]
I think giving consumers the choice could be the right move given that force press activates on a fairly light touch

**Comment by markbrown4** [comment]
It seems to me that a fix for an edge case(wanting force touch within a draggable) made the majority case of simply dragging on ios much worse.

I feel like the force touch code cancellation should be removed, made opt-in or left in userland completely.  If someone is dragging on a drag handle the intention is to drag.

**Comment by alexreardon** [comment]
The intention is to play as nicely as possible with existing browser
interactions

**Comment by alexreardon** [comment]
I am keen to get this right. I hope to explore it soon

**Comment by alexreardon** [comment]
Perhaps a shouldRespectForceTouch prop?

**Comment by alexreardon** [comment]
Fixed by #1148. Available in [10.1](https://github.com/atlassian/react-beautiful-dnd/releases/tag/v10.1.0)

**Comment by markbrown4** [comment]
Thanks Alex 🙏
